llmfoundry/__init__.py

```diff
@@ -13,19 +13,38 @@
 
 # Filter out Hugging Face warning for not using a pinned revision of the model
 hf_dynamic_modules_logger = logging.getLogger(
-    'transformers.dynamic_module_utils')
+    'transformers.dynamic_module_utils',
+)
 new_files_warning_filter = SpecificWarningFilter(
-    'A new version of the following files was downloaded from')
+    'A new version of the following files was downloaded from',
+)
 
 hf_dynamic_modules_logger.addFilter(new_files_warning_filter)
```

Comment:
can we just call it logger and make it 1 line instead of indenting?

Reply:
Done